### DISCUSSION TITLE
FIDO MDS Integration

### DISCUSSION THREAD

**Discussion by randheersingh437** [webauthn4j/webauthn4j#428] [Q&A] [answered]
FIDO MDS Integration: 
Do we have any standard way of refreshing the access token?
 Based on what I read, access token is sent on email on request. However this can be a maintenance nightmare for the enterprise application. As there is no other endpoint available to check the validity of access token, admin will only know, upon runtime failure.
Anu guidance from your side about how to manage access token is deeply appreciate.

**Comment by ynojima** [comment, marked as answer]
Sorry I don't have answer for this question. It is better to ask on `fido-dev@fidoalliance.org`.
https://fidoalliance.org/developers/resources/
